src/Chess/Game/ChessGame.php

```diff
@@ -764,6 +767,9 @@ public function inCheck($color)
     {
         $ret = array();
         $kingLoc = $this->_pieces[$color . 'K'];
+        if (!$kingLoc) {
```

Comment:
mmmm... I wonder if we need these because of a faulty test. Chess games must have 2 kings, if I understand things correctly, in the test at least one is not in the board. Returning early seems fine though. If there's no king, there's no check :smile: